src/io/jesi/backpack/miscellaneous.cljc

```diff
@@ -30,6 +30,7 @@
 (defn ->uuid-or-not [id]
   (or (->uuid id) id))
 
+;TODO move to collection
 (defn collify
   "Puts value `v` in a vector if it is not a collection. Returns `nil` if no value"
   ([] nil)
```

Comment:
Just move it. everything should be referencing the root backpack ns so it shouldn't break anything.

Reply:
good point